handbook/demand/demand.rituals.yml

```diff
@@ -43,6 +43,13 @@
   description: "Every release cycle, upload the ☁️🌈 Sprint demos video to YouTube"
   moreInfoUrl: "https://fleetdm.com/handbook/demand#upload-to-youtube"
   dri: "drewbakerfdm"
+  -
```

Comment:
check the syntax vs other rituals

Reply:
Ahh, might've been an out of place space. Should be fixed.